### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by SerhiiSoloviov**

### CODE REVIEW

**Review by AnatoliiDomin — APPROVED** [review]
Approved!

File: cypress/e2e/checkout.cy.js [hunk 1]
```diff
@@ -1,11 +1,45 @@
...
 
+  it('should automate flow using POM', () => {
+    homePage.clickOnCategory('Laptops');
+    homePage.clickOnProduct('Sony vaio i7');
+    cart.clickOnBTNaddToCart('Add to cart');
+    alert.assertAllert('Product added');
+    homePage.clickOnLink('Cart');
+    cart.productIsInTheCart('Sony vaio i7');
+    cart.clickOnPlaceOrder('Place Order');
+    cart.nameField(testData.name, { force: true });
```
Comment (djtyrf312) [inline cypress/e2e/checkout.cy.js:35]: it is hard to read this method. What does it mean cart.nameField()? то fill it or check it?

File: cypress/e2e/checkout.cy.js [hunk 2]
```diff
@@ -1,11 +1,45 @@
...
+    cart.countryField(testData.country, { force: true });
+    cart.cityField(testData.city, { force: true });
+    cart.creditCardField(testData.card, { force: true });
+    cart.monthField(testData.month, { force: true });
+    cart.yearField(testData.year, { force: true });
+    cart.clickOnPurchase('Purchase');
+    cart.enteredDataSsShownOnModal(testData.card, testData.name);
```
Comment (djtyrf312) [inline cypress/e2e/checkout.cy.js:42]:
```suggestion
    cart.assertOrderData(testData.card, testData.name);
```

File: cypress/e2e/checkout.cy.js [hunk 3]
```diff
@@ -1,11 +1,45 @@
...
+    cart.clickOnOk('OK');
```
Comment (djtyrf312) [inline cypress/e2e/checkout.cy.js:43]:
```suggestion
    cart.clickOkButton();
```

**Review by djtyrf312 — CHANGES_REQUESTED** [review]
check comments

**Pull request closed without merging**